src/main/java/stringCalculator/UserInputValue.java

```diff
@@ -15,16 +15,18 @@
         String input = scanner.nextLine();
         List<String> userInputTokens = Arrays.asList(input.split(" "));
 
-        validateBlankCarName(userInputTokens);
+        validateBlank(userInputTokens);
         validateEmptyNumber(userInputTokens);
         validateEmptyOperation(userInputTokens);
 
         return userInputTokens;
     }
 
-    public void validateBlankCarName(List<String> userInputTokens) {
-        if (userInputTokens.size() == 0) {
-            throw new IllegalArgumentException(BLANK_CARNAME);
+    public void validateBlank(List<String> userInputTokens) {
+        for (String userInputToken: userInputTokens) {
+            if (userInputToken == "") {
```

Comment:
```suggestion
                        if (Objects.equals(userInputToken, "")) {
```
대상값을 비교하기 위해 `equals`를 사용해보는건 어떨까요?